index.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>HTML Quiz | Chase Ramsey</title>
+	<link rel="stylesheet" type="text/css" href="css/normalize.css">
+	<link rel="stylesheet" type="text/css" href="css/stylez.css">
+	<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,400italic,300italic' rel='stylesheet' type='text/css'>
+</head>
+<body>
+
+<header id="main-header" class="clearfix">
+<h1><a href="#"><span class="u_logo">YOU</span>nicorn</a></h1>
+	<nav>
+		<ul>
+			<li><a href="#" alt="Contact">Contact</a></li>
+			<li><a href="#" alt="About">About</a></li>
+			<li><a href="#" alt="Home">Home</a></li>
+		</ul>
+	</nav>
+</header>
+
+<!-- This <div> id #wrapper will group all of the main content in the body. Its direct children are the article cards.
+		 "#wrapper: padding-top" is set to 80px to push the #wrapper out from under the header. 														-->
+	<div id="wrapper" class="clearfix">
+
+	<!-- Start <article> 1 -->
+	<article>
+		<header class="card_title">
+			<h1>The Starchaser</h1>
```

Comment:
Consider using the only one `h1` per page. Using multiple `h1`s can mess with screen readers and other tools.